extractors-clipbyshape/README.md

```diff
@@ -0,0 +1,34 @@
+# Plot clipping by shapefile extractor
+
+Clip GeoTIFF files according to shapefile geometry
+
+## Overview
+
+Clipping is triggered by a shapefile being added to a dataset.
+The clipped files are placed into a new dataset and organized by plot.
+Image pixel size is consistency maintained for all clipped images on a per-plot basis.
+
+A date value is used to organize the clipped files.
+By deffault the local machine date for when the clipping was started is used.
+To force the date to a certain value, include it in the shapefile name with a leading ' - ' separator.
+A trailing ' - ' separator is needed if the date is not the last part of the filename before the file extension.
+The date must use a four digit year format and either come first or last.
+The day and month need to be two digits each.
+The placement of month and day in relation to each other is ignored: DD-MM-YYYY is as good as MM-DD-YYYY and YYYY-DD-MM.
```

Comment:
If DD-MM-YYYY is as good as MM-DD-YYYY how would you differentiate 05-10-2018 being May 10th vs. October 5th?

Reply:
Can we please ISO-8601 standard YYYY-MM-DD

Reply:
The documentation has been updated